ASSEMBLY.md

```diff
@@ -124,6 +126,13 @@ Time to learn Rust and get hacking on the BIOS and the OS! That's what the Neotr
 
 You can make one out of a Raspberry Pi Pico. See <https://github.com/rp-rs/rp2040-project-template/blob/main/debug_probes.md> for more details.
 
+For Linux Mint, the udev-rules also needs to be updated for the SWD debug probe. The file `/etc/udev/rules.d/99-cmsis-dap.rules` should include:
+
+```
+# cafe:4005 RP2040 Pico with CMSIS-DAP
+SUBSYSTEM=="usb", ATTR{idVendor}=="cafe", ATTR{idProduct}=="4005", MODE:="0666"
+```
```

Comment:
Add a note here to adjust the rule according to the VID and PID of the programmer being used.

Reply:
Done